script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -30,13 +30,13 @@ All right. In that case, come by the tea ceremony club after school. You're more
 I feel embarrassed when she says such things with a soothing smile.
 Unable to give a witty reply, all I can muster is a weak 'yes'.
 <Page11>
-"It's a promise then. Between you and me, we have a surplus of snacks purchased with club funds. It's not something that stays fresh long, so I need a healthy boy to help out."
+"It's a promise, then. Between you and me, we have a surplus of snacks purchased with club funds. It's not the kind of thing that stays fresh long, so I need a healthy boy to help out." //Same edit as the previous file, keep both or neither.
 "I-Is that so? Well, if my schedule permits, I'll certainly take you up on that offer.
 Although, getting taken in by food is something a cat would do."
 <Page12>
 Her smile is so friendly that I can't help but joke a little.
 <Page13>
-The Senpai with a green ribbon replies,
+The Senpai with a green ribbon replies, //Maybe use this for the "ribbon-wearing" bit from the last file? I think a quick check between these two scenes would be good, this one is essentially the other one minus Shiki being horny.
```

Comment:
I'm personally not a fan of using "a/the + senpai/sensei etc."
I would replace with `Senpai, who was wearing a green ribbon, replies,`
Or `The upperclassman with a green ribbon replies`

And if we're keeping "the senpai", then maybe without a capital S?

Reply:
"Upperclassman" flows pretty well here, I'll put that in for now.